scripts/run.sh

```diff
@@ -43,8 +43,8 @@ MODE=${3:-tutorial}
 REPOSITORY=rgardler
 CONTAINER_NAME=simdem_$FLAVOR
 SCRIPTS_VOLUME=${CONTAINER_NAME}_scripts
-AZURE_VOLUME=azure_data
-SSH_VOLUME=ssh_data
+AZURE_VOLUME=${AZURE_VOLUME:-azure_data}
```

Comment:
This looks like a good change in principle, but I'm not sure if this is the best approach... lets discuss...

In my case I have two data volumes, "azure_data" and "ssh_data", these are reused across many different containers. I have them available in my private docker registry, do all that is needed to work from a new machine is a docker pull of those volumes.

In a single machine environment mounting ".azure" and ".ssh" would make more sense, I agree.

Perhaps the best approach is to cater for the (more common?) single client machine use case, thus making the defaults ".ssh" and ".azure".

If we do this should we use command line parameters to change the default, introducing params 4 and 5 (i.e. `${4:-~/.azure})

ASIDE (not a blocker for this PR) with 4-5 params we really auto put command line options in (e.g. `--ssh_dir=~/.ssh`)

If you agree please make the change to this PR, if not lets disuss